src/sentry/integrations/tasks/sync_status_inbound.py

```diff
@@ -270,22 +278,27 @@ def sync_status_inbound(
                 if not created:
                     resolution.update(datetime=django_timezone.now(), **resolution_params)
 
-            try:
-                default_user_id = str(organizations[0].get_default_owner().id)
-            except IndexError:
-                logger.exception("Error getting default user")
-                default_user_id = "<unknown>"
+            issue_resolved.send_robust(
+                organization_id=organization_id,
+                user=None,
+                group=group,
+                project=group.project,
+                resolution_type=provider.key,
+                sender=f"resolved_with_{provider.key}",
+            )
+
             analytics.record(
                 "issue.resolved",
                 project_id=group.project.id,
-                default_user_id=default_user_id,
+                default_user_id="Sentry Jira",
```

Comment:
Turns out `default_user_id` can't be `None`, so made rando string to at least make it clear it's Jira resolving the issue